terraform-ecs-fargate-service/service.tf

```diff
@@ -1,69 +1,196 @@
 
-resource "aws_ecs_task_definition" "task_definition_api" {
+resource "aws_ecs_task_definition" "service" {
   family                   = var.service_name
   execution_role_arn       = var.task_execution_role_arn
   task_role_arn            = var.task_role_arn
   requires_compatibilities = ["FARGATE"]
   network_mode             = "awsvpc"
-  cpu                      = 256
-  memory                   = 512
```

Comment:
whut, it was harcoded like this? Or was this a default in case it wasn't specified in the service?

Reply:
It was default, being rewritten in the deploy script: https://github.com/FigurePOS/fgr-service-3rd-party-aggregators/blob/c974785393363b6478aa497e67a3ff6729220dc6/scripts/deploy.sh#L308-L309

But I have one question to this. There are 9 services that have it defined in SSM parameters (like https://github.com/FigurePOS/fgr-message-router/blob/ff44255f7c2492023d8a8c96e0f793817f2ac97b/tf/service.tf#L40-L50) and the other 11 have it hardcoded (like https://github.com/FigurePOS/fgr-service-business-config/blob/dbdb3b4370ef2180288f53fcf470bfca6dce8f5e/scripts/deploy.sh#L17-L18). What's the right way? 🙂 Trubi told me once that it doesn't make sense to him to commit in terraform some values that are being changed often (it was regarding number of tasks maybe), so maybe this was the same idea, but imho the memory and cpu is not changed quite often. 🙂 

(And there are some `memory_reservation` parameters in SSM that are not used anywhere in the code: https://github.com/search?q=org%3AFigurePOS%20NOT%20is%3Aarchived%20memory_reservation&type=code These can be deleted?)

Reply:
What would you prefer? I don't have any issue having it in the deploy script... I don't see this as something that changes often but I don't really know what would be the benefit of having it in ssm 🤔

Reply:
🤔 I think that Trubi was saying something like that he would like to see some management UI that would allow you to see all SSM parameters for each service and allow you to change them directly. Maybe that was the reason because I wouldn't see added value in managing them in the SSM console where it is one big mess. 🙂

Reply:
Hmm, yeah I don't think that is happening any time soon + I think it is some nice to have but if you see value in that feel free to keep it in ssm